Fixes FWLinks for PS7 online help documents

davidsmatlak opened the pull request:
<!-- Anything that looks like this is a comment and can't be seen after the Pull Request is created. -->

# PR Summary

Updates the FWLinks for online help documents.
<!-- Summarize your PR between here and the checklist. -->

## PR Context

Fixes broken links so that the FWLinks open the correct online help document. 
<!-- Provide a little reasoning as to why this Pull Request helps and why you have opened it. -->

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): <!-- Experimental feature name(s) here -->
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: <!-- Number/link of that issue here -->
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

davidsmatlak committed:
fix fwlinks

adityapatwardhan commented on test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv:
```diff
@@ -1,98 +1,98 @@
 Enable-ScheduledJob,https://go.microsoft.com/fwlink/?LinkID=223926
+Enter-PSHostProcess,https://go.microsoft.com/fwlink/?linkid=2096580
+Exit-PSHostProcess,https://go.microsoft.com/fwlink/?linkid=2096583
+Expand-Archive,https://go.microsoft.com/fwlink/?linkid=2096769
```
Please remove this change till the module has shipped with the update.

davidsmatlak replied:
Removed `Expand-Archive` from the CSV file.

adityapatwardhan commented on test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv:
```diff
@@ -1,98 +1,98 @@
-ConvertTo-Json,https://go.microsoft.com/fwlink/?LinkID=217032
+Add-LocalGroupMember,https://go.microsoft.com/fwlink/?LinkId=717987
+Clear-RecycleBin,https://go.microsoft.com/fwlink/?LinkId=2109377
+Compress-Archive,https://go.microsoft.com/fwlink/?linkid=2096473
```
Please remove this change till the module has shipped with the update.

davidsmatlak replied:
Removed `Compress-Archive` from the CSV file.

davidsmatlak committed:
removed archive cmdlets

davidsmatlak commented:
Removed `Expand-Archive`  and `Compress-Archive` from the V3Cmdlets.csv file.

Fixed conflict in `TestConnectionCommand.cs` by accepting the incoming change and then updating the FWlink.

adityapatwardhan commented:
PR created to update `Compress-Archive` and `Expand-Archive` cmdlet help fwlinks. https://github.com/PowerShell/Microsoft.PowerShell.Archive/pull/82
